src/emmodel/model.py

```diff
@@ -129,6 +152,18 @@ def plot_time_trend(ax: plt.Axes, df: pd.DataFrame,
     return ax
 
 
+def simulate_uncertainty(coef, vcov, parameters, data, num_draws):
+    lst_pred = []
+    for _ in range(num_draws):
```

Comment:
This loop might be slow, I think it will be better to use `np.random.multivariate_normal` with  `size` option. And then write a list comprehensive to create `lst_pred`.